src/data.js

```diff
@@ -1786,6 +1786,29 @@ module.exports = [
       'SQL',
     ],
   },
+  {
+    name: 'Mostafa Hosseini',
+    description: 'Fullstack Developer',
+    url: 'https://mostafa-hosseini.me/uses',
```

Comment:
This isn't a uses page yet, please let us know when it is and I'll get this merged.

Reply:
I added /uses/ page, please review